src/content/en/updates/2013/07/Alpha-transparency-in-Chrome-video.md

```diff
@@ -20,14 +21,14 @@ description: WebM just added alpha transparency, and support for it has landed i
   </iframe>
 </div>
 
-Chrome Canary now supports video alpha transparency in WebM.
+Chrome now supports video alpha transparency in WebM.
 
 In other words, Chrome takes the alpha channel into account when playing '[green
-screen](https://en.wikipedia.org/wiki/Chroma_key)' videos encoded to WebM with
+screen](https://en.wikipedia.org/wiki/Chroma_key)' videos encoded to WebM ([VP8] and [VP9]) with
```

Comment:
Do VP8 and VP9 need a link here?

Reply:
Do you suggest we remove them completely or move them?
I think it's useful when looking at the timeline.

Reply:
I just wasn't sure about the square brackets :). (Apologies if I'm missing something obvious.)

Reply:
`[]` allow me to not inline URLs. See at the bottom of the article.
Live preview is available at https://pr-7986-dot-web-central.appspot.com/web/updates/2013/07/Alpha-transparency-in-Chrome-video by the way